keyboards/alps64/keymaps/zyber/keymap.c

```diff
@@ -0,0 +1,31 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```


Also, if you have `#include QMK_KEYBOARD_H` in your zyber.h file, you could just include that here instead.